cost/aws/old_snapshots/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.17
+
+- added changes from [wiki](https://confluence.flexera.com/pages/viewpage.action?spaceKey=rightscale&title=Usage+Recommendations%3A+Waste+reduction)
```

Comment:
This is an internal document and should not be referenced in a public ChangeLog.